lib/ice_cube/parsers/ical_parser.rb

```diff
@@ -75,7 +86,7 @@ def self.rule_from_ical(ical)
         when "BYYEARDAY"
           validations[:day_of_year] = value.split(",").map(&:to_i)
         when "BYSETPOS"
-          # noop
```

Comment:
I find it kinda funny (/weird) that they knew about `BYSETPOS`, but decided to just silently ignore it. Why wouldn't they let it fall into the `else` to raise as being unsupported 🤷

Reply:
😄, Indeed!